docs/reference/setup/install/windows.asciidoc

```diff
@@ -369,11 +435,16 @@ The `.msi` can also upgrade Elasticsearch using the command line.
 [IMPORTANT]
 ===========================================
 A command line upgrade requires passing the **same** command line properties as
-used at first install time; the Windows Installer does not remember these properties.
+used at first install time; the Windows Installer does not remember these properties. 
 
 For example, if you originally installed with the command line options `PLUGINS="x-pack"` and
 `LOCKMEMORY="true"`, then you must pass these same values when performing an
 upgrade from the command line.
+
+The **exception** to this is `INSTALLDIR` (if originally specified), which must be a different directory to the 
+current installation. 
+If setting `INSTALLDIR`, we recommend that the final directory in the path be the version of Elasticsearch e.g.
+++C:\Program Files\Elastic\Elasticsearch\{version}++
```

Comment:
Not just recommended but currently we have to mandate the installer ends in the version when running the silent install otherwiae the install will fail. THE UI can cope with this though.

Reply:
I think the installer should only mandate that the directory for the upgrade version is different to the existing version, and not a child directory of the existing version.